src/module/apps/compendium-browser/index.ts

```diff
@@ -310,10 +310,8 @@ class CompendiumBrowser extends Application {
         }
 
         // Sorting
-        if (this.initialFilter.orderBy) {
-            currentTab.filterData.order.by = this.initialFilter.orderBy;
-            currentTab.filterData.order.direction = this.initialFilter.orderDirection ?? "asc";
-        }
+        currentTab.filterData.order.by = this.initialFilter.orderBy ?? "name";
```

Comment:
is this meant to be name?

Reply:
It's a safe default if not set since not all item types have levels